index.py

```diff
@@ -2,36 +2,47 @@
 import os
 import yaml
 
-from my_lambda_package.utility import Utility
+from utils.helpers import Helpers
+from utils.config import configuration
 
 
-logging.basicConfig()
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+def setup_logging(request_id, level=logging.INFO):
+    """Creates the logging formatter.
 
+    Args:
+        request_id: (str) The id of the execution context (i.e. the Lambda execution ID).
+        level: logging level to use.
+    """
+    logger = logging.getLogger()
+    logger.info('Setting up logging')
+    console_handler = logging.StreamHandler()
+    formatter = logging.Formatter(
+        '[%(levelname)s] %(asctime)s {0} [%(module)s:%(lineno)d]: %(message)s'.format(request_id))
+    console_handler.setFormatter(formatter)
 
-def _load_config(filename='config.yaml'):
-    """Loads the configuration file."""
-    with open(os.path.abspath(os.path.join(os.path.dirname(__file__), filename)), 'r') as f:
-        config = yaml.load(f)
-        logger.info('Loaded config: {0}'.format(config))
-    return config
+    logger.handlers = []  # Get rid of any default handlers (Lambda apparently adds one).
+    logger.addHandler(console_handler)
+    logger.setLevel(level)
+    return logger
 
 
 def handler(event, context):
     """Entry point for the Lambda function."""
-    config = _load_config()
+    logger = setup_logging(context.aws_request_id)
```

Comment:
Is there a good way to invoke the function with debug logging level? I'm not super familiar with that part of Lambda.